types/viewRaidData.d.ts

```diff
@@ -7,7 +7,7 @@ interface UserStats {
   commits: number
 }
 
-interface ViewRaidData {
+interface RaidStats {
```

Comment:
Should probably rename the file to `raidStats.d.ts` while we are at it :thinking:

Reply:
on it